doc/source/conf.py

```diff
@@ -16,6 +16,7 @@
 
 import os
 import sys
+from satpy import __version__
```

Comment:
This should probably be imported after the `sys.path` line below it. This would then allow someone to build the docs from a version of satpy that they don't have installed.

Reply:
ok, I can change that